src/modules/GameConstants.ts

```diff
@@ -1818,3 +1818,7 @@ export enum ExtraAchievementCategories {
     sevii,
     magikarpJump,
 }
+export enum DayCycleStartHours {
+    dawn = 6,
+    dusk = 17,
+}
```

Comment:
Make this a `Record<DayCyclePart, number>` and fill it in for all DayCycleParts, so we aren't relying on dusk and dawn being 1 hour long.